### DISCUSSION TITLE
Question: Week 7 not appearing in desktop version or changelog.md

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1529] [Q&A]
## What is your question?
Week 7, "Tankman", has been released, but I do not see it appearing anywhere in the game.

**Comment by zatrit** [comment]
Tankman is Newgrounds exclusive temporarily

**Comment by ghost** [comment]
So no week 7 ever

**Comment by shanepaton** [comment]
I don't know...
What sucks is that Haxefixel never loads on the Newgrounds version even with a good PC virtually making Week 7 unplayable.

**Comment by NotHyper-474** [comment]
> I don't know...
> What sucks is that Haxefixel never loads on the Newgrounds version even with a good PC virtually making Week 7 unplayable.

Actually it just takes too long. And also sometimes it freezes on a percentage and you have to refresh the page to fix
